packages/web-components/src/components/button/button.ts

```diff
@@ -94,6 +94,7 @@ class DDSButtonExpressive extends FocusMixin(StableSelectorMixin(LitElement)) {
     const { _handleSlotChange: handleSlotChange } = this;
     return html`
       <slot @slotchange="${handleSlotChange}"></slot>
+      <p class="${prefix}--btn--hidden" aria-hidden="true">${'(' + this.href + ')' /* eslint-disable-line prefer-template */}</p>
```

Comment:
```suggestion
      <p class="${prefix}--btn--hidden" aria-hidden="true">(${this.href})</p>
```

Concatenation like this may not be needed after all since this is a big string after all, the parentheses will be a normal part of the markup and then the href will be placed inside properly.